sandbox/modules/custom-codecs/build.gradle

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+apply plugin: 'opensearch.opensearchplugin'
+apply plugin: 'opensearch.yaml-rest-test'
+
+opensearchplugin {
+  name 'custom-codecs'
+  description 'A plugin that implements zstd (with and without dictionary) and lz4 (native) compression algorithms.'
```

Comment:
Maybe "A plugin that implements custom codecs, including zstd ..."

Reply:
Description fixed. 

Regarding the runtime permission, jdk [limits us](https://github.com/openjdk-mirror/jdk7u-jdk/blob/master/src/share/classes/java/security/BasicPermission.java#L103-L119) from qualifying `loadLibrary.*` further. I see that [ZSTD snapshotting compression](https://github.com/opensearch-project/OpenSearch/pull/2996) #2996 has the same issue.